.github/CODEOWNERS

```diff
@@ -3,5 +3,3 @@
 # global
 * @hypertrace/build-tooling
 
-# GH action
-.github/ @aaron-steinfeld @jbahire @kotharironak
```

Comment:
This doesn't really serve a purpose any more, so removed the action codeowners - but will still need @kotharironak or @JBAhire to approve this one.